examples/supabase.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * @title Connect to Supabase
+ * @difficulty intermediate
+ * @tags cli, deploy
+ * @run --allow-net --allow-env <url>
+ * @resource {https://supabase.com/docs/reference/javascript/installing} supabase-js docs
+ * @resource {https://jsr.io/@supabase/supabase-js} supabase-js on jsr.io
+ * @group Databases
+ *
+ * You can connect to a Supabase database using the `supabase-js` library.
+ */
+
+// Import the createClient function from jsr supabase-js package
+import { createClient } from "jsr:@supabase/supabase-js@2";
+
+// Create a single supabase client for interacting with your database
+const supabase = createClient(
+  Deno.env.get("SUPABASE_URL")!,
+  Deno.env.get("SUPABASE_KEY")!,
+);
+
+// Insert data into the countries table
+export async function insertCountryData(
```

Comment:
The deno examples are meant to be standalone scripts that users can easily run and see the output directly. It would be great if you could either invoke the functions right after their definitions or move the code outside of the functions, so the output is visible immediately when the script is run. This would make it more user-friendly. Thanks!

Reply:
Hey, I made the changes as requested, I thought about using IIFEs but in general it would add more complexity to the code, so instead I moved the code outside and use a variable to receive and show the values. Now it's more like a script and reduced in size.